Package.swift

```diff
@@ -87,7 +87,7 @@ let package = Package(
 )
 
 #if compiler(>=6)
-  for target in package.targets where target.type != .system {
+  for target in package.targets where target.type != .system && target.type != .test {
```

Comment:
I added this temporarily so that we don't get flooded with warnings in test targets, but maybe we should keep this around until we've addressed all/most warnings in TCA proper, and then we can later concentrate on tests?